src/control/server/storage_nvme.go

```diff
@@ -242,6 +243,11 @@ func (n *nvmeStorage) Format(i int, results *(types.NvmeControllerResults)) {
 	}
 
 	switch srv.BdevClass {
+	case bdFile:
+		log.Debugf("bdev file format successful (%s)\n", pciAddr)
+
+		addCretFormat(pb.ResponseStatus_CTRL_SUCCESS, "", msgBdevClassIsFile)
+		n.controllers = loadControllers(cs, ns)
```

Comment:
@tanabarr Could you please verify that that I address the merge conflict correctly? I wasn't clear whether msgBdevClassIsFile should be passed as the 2nd args and whether loadControllers() should be called for bdev file. Thanks in advance.